server/auth/registrar.go

```diff
@@ -152,21 +153,45 @@ func (auth *AuthManager) handleNotifyFee(conn comms.Link, msg *msgjson.Message)
 		}
 	}
 
+	auth.feeWaiterMtx.Lock()
+	if _, found := auth.feeWaiterIdx[acctID]; found {
+		auth.feeWaiterMtx.Unlock() // cannot defer since first TryFunc is synchronous and may lock
+		return &msgjson.Error{
+			Code:    msgjson.FeeError,
+			Message: "already looking for your fee coin, try again later",
+		}
```

Comment:
Or maybe `return nil`?  This is probably just a duplicate notify_fee request and sending an error might confuse dexc.